frontend/src/app/pages/exercises/exercise/shared/transfer-overview/transfer-time-input/transfer-time-input.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="btn-group btn-group-sm text-nowrap">
+    <button
+        class="btn btn-small btn-outline-secondary"
+        title="Um 5 Minuten verkürzen"
+        (click)="addTransferTime(-5 * 60 * 1000)"
+    >
+        -5min
```

Comment:
```suggestion
        -5&nbsp;min
```

For all such units (see e.g. [here](https://tu-freiberg.de/fakultaet2/exphys/lehre/formatierungstipps#:~:text=Zwischen%20Zahl%20und%20Einheit%20steht,oder%20%5C%2C%20(kurz).)).